src/functions/addProvisionalTechRecord.ts

```diff
@@ -0,0 +1,42 @@
+import TechRecordsDAO from "../models/TechRecordsDAO";
+import TechRecordsService from "../services/TechRecordsService";
+import HTTPResponse from "../models/HTTPResponse";
+import ITechRecordWrapper from "../../@Types/ITechRecordWrapper";
+import ITechRecord from "../../@Types/ITechRecord";
+import {ONLY_DIGITS_AND_NUMBERS} from "../assets/Enums";
+
+const addProvisionalTechRecord = (event: any) => {
+  const techRecordsDAO = new TechRecordsDAO();
+  const techRecordsService = new TechRecordsService(techRecordsDAO);
+
+  const techRec: ITechRecord[] = event.body ? event.body.techRecord : null;
+  const msUserDetails = event.body ? event.body.msUserDetails : null;
+  const vin = event.body ? event.body.vin : null;
+  const sysNum = event.body ? event.body.systemNumber : null;
```

Comment:
systemNumber should be on the pathParameters and also the vin is not used, can be removed from the payload